### ISSUE TITLE
database error on changing events end time

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#92]
```
What steps will reproduce the problem?
1.switch an events end time 
2.
3.

What is the expected output? What do you see instead?
In logs/sql i expect an 
 UPDATE event statement 
but i found 
 SELECT * FROM users WHERE user_id='unknown'

What version of the product are you using? On what operating system?
trunk-r-3878, jqueryui 1.8.2.1, SLES11SP1, postgresql 

Please provide any additional information below.
In calendar.php I add in the getUser function two lines 1949/50
  if (!defined($id))
      return array();

```

Original issue reported on code.google.com by `conradsh...@googlemail.com` on 12 Aug 2010 at 10:08

**Comment by GoogleCodeExporter** [comment]
```
Did this fix the issue?
```

Original comment by `rol...@liebl.ath.cx` on 12 Aug 2010 at 11:30
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
yes, thank you
```

Original comment by `conradsh...@googlemail.com` on 12 Aug 2010 at 12:07

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 12 Aug 2010 at 2:21

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3878.1.
```

Original comment by `rol...@liebl.ath.cx` on 14 Aug 2010 at 4:53
- Changed state: **Fixed**
